src/datamodel/game/quest_objective.rs

```diff
@@ -0,0 +1,4 @@
+pub struct QuestObjective {
+    pub num_players: u32,
+    pub required_successes: u32,
+}
```

Comment:
why having this apart? could this struct go into the board module?
Like `Character` and `CharactersName`.

Reply:
Sounds good. I am still not 100% sure what's the most rustacean way of organising the code, I'm used to Java where there best practise is to have one file per high level class.

Reply:
here you have the modules doc from the book:
https://doc.rust-lang.org/book/ch07-05-separating-modules-into-different-files.html
In this case I think that `QuestObjective` should be inside of `board.rs` because as `CharacterName` is inside `character` module.